packages/docs/src/components/DocSiteNavigation.tsx

```diff
@@ -136,22 +165,51 @@ const DocSiteSidebar = ({ isMobileNavOpen, location }: DocSiteNavProps) => {
   const navItems: VerticalNavItemProps[] = formatNavData(data?.allFile?.group);
 
   return (
-    <nav
-      className={classnames('ds-l-md-col--3 ds-u-padding--2 ds-u-fill--white c-sidebar', {
-        'c-sidebar--open': isMobileNavOpen,
+    <div
+      className={classnames('ds-l-md-col--3 ds-u-padding--0 ds-u-md-padding--2 c-navigation', {
```

Comment:
I think the side bar width should be fixed rather than a 3/12 grid column. It looks too wide on some screens and too narrow on others. What do you think, @line47?

Reply:
I agree, fixed width for the left nav